app/views/owners/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_for(@owner) do |f| %>
+  <%= f.label  :full_name %><br>
+  <%= f.text_field :first_name, placeholder: "First name..." %>
+  <!-- < f.label  :middle_name %> -->
+  <%= f.text_field :middle_name, placeholder: "Middle name..." %>
+  <!-- < f.label  :last_name %> -->
+  <%= f.text_field :last_name, placeholder: "Last name..." %><br>
+  <%= f.label  :address %><br>
+  <%= f.text_field :address, placeholder: "Street name/number..."%>
+  <!-- < f.label  :city %> -->
+  <%= f.text_field :city, placeholder: "City..."%>
+  <!-- < f.label  :state %> -->
+  <%= f.text_field :state, placeholder: "State..." %>
+  <!-- < f.label  :zip %> -->
+  <%= f.text_field :zip, placeholder: "Zipcode..."%><br>
+  <%= f.label  :years_lived_at_home %>
```

Comment:
Thanks for starting to work on these form changes! You need to use slightly different syntax for this, though. Rails only knows about `:years`; it doesn't know anything about `:years_lived_at_home`. You can use the same object and just pass an [additional string variable](https://apidock.com/rails/ActionView/Helpers/FormHelper/label):

```suggestion
  <%= f.label  :years, "Years lived in your current home" %>
```

Reply:
Oh, okay. I'll modify it. Thank you!